src/celeritas/phys/PhysicsData.hh

```diff
@@ -265,7 +268,8 @@ struct PhysicsParamsScalars
                    == static_cast<bool>(fixed_step_action))
                && lambda_limit > 0 && range_factor > 0 && range_factor < 1
                && safety_factor >= 0.1
-               && step_limit_algorithm != MscStepLimitAlgorithm::size_;
+               && step_limit_algorithm != MscStepLimitAlgorithm::size_
+               && spline_eloss_order > 0;
```

Comment:
Move this check to right before `lambda_limit > 0` so that the ordering is the same as the variables.